data/plugins/CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+## How to contribute
+
+If you wish to recommend a plugin or framework not included in the list yet:
+
+1. create a `.json` file in [data/plugins/myplugin.json](myplugin.json) including plugins credentials as
+```
+{
+  "name": "myplugin",
+  "description": "an awesome neovim plugin",
+  "link": "https://neovim.io/",
+  "tags": [
+    "awesomeness",
+    "coolness",
+    "lua"
+  ],
+  "examples": []
+}
+```
+
+2. (optional) (make sure that the repository fulfils the below criteria)
```

Comment:
Only criteria I would add is to just check existing tags

Reply:
Added this comment in af87fabbc3e7fef4f068b17fa5e00b879ceff55f